webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageActions.js

```diff
@@ -0,0 +1,24 @@
+import { foremanUrl } from '../../../../foreman_tools';
+
+import {
+  REGISTRATION_COMMANDS_DATA,
+  REGISTRATION_COMMANDS_OS_TEMPLATE,
+  REGISTRATION_COMMANDS,
+} from '../constants';
+
+export const dataAction = params => ({
+  key: REGISTRATION_COMMANDS_DATA,
+  url: foremanUrl('/hosts/register/data'),
+  params,
+});
```

Comment:
you can call the API method right here, e.g:
```js
export const dataAction = params => get({
  key: REGISTRATION_COMMANDS_DATA,
  url: foremanUrl('/hosts/register/data'),
  params,
});
```
and dispatch it from `index.js`, I think it will make more sense